src/apiClient.js

```diff
@@ -176,9 +176,7 @@ class ApiClient {
         }
 
         if (values.length) {
-            const auth = `MediaBrowser ${values.join(', ')}`;
-            //headers.Authorization = auth;
-            headers['X-Emby-Authorization'] = auth;
+            headers['Authorization'] = `MediaBrowser ${values.join(', ')}`;
```

Comment:
You should be able to just use the `.` notation since it doesn't contain any weird characters :upside_down_face: 

```suggestion
            headers.Authorization = `MediaBrowser ${values.join(', ')}`;
```

Reply:
I know. Using string highlights it, makes it special.